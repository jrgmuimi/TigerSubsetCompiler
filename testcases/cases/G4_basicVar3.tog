let 
    var basicVar1: int
    var basicVar2: int
    var basicVar3: int
    var basicVar4: int
    var basicVar5: int
    var basicVar6: int
    var basicVar7: int
    var basicVar8: int
    var basicVar9: int

in
    basicVar1 := 540;
    basicVar2 := 440;
    basicVar3 := 1;
    basicVar4 := 2;
    basicVar5 := 3;
    basicVar6 := 2;
    basicVar7 := 2;
    basicVar8 := 11111111;
    basicVar9 := 0*0;
    printint(basicVar1);
    printint(basicVar2);
    printint(basicVar3);
    printint(basicVar4);
    printint(basicVar5);
    printint(basicVar6);
    printint(basicVar7);
    printint(basicVar8);
    printint(basicVar9)
end