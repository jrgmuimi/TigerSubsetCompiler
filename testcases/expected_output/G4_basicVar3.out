MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

54044012322111111110
